src/index.js

```diff
@@ -7,7 +7,8 @@ import { BrowserRouter } from "react-router-dom";
 
```

Comment:
It looks like we need to get a fresh fetch from the main branch as this pull PR (pull request) does not have the latest merge to the main. What we see specifically is the redux store is missing.